wikiman.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 mediawikiVersion: "1.39"
-mediawikiRepoRef: 3406e1b7d014d22f2a68237ebf447a88b5ac74ed
+mediawikiRepoRef: ec2d725f9876b663f5ed736340f54814c023431c
```

Comment:
I wonder if we would rather remove this? I think we'd be fine always getting the latest 1.39 version.